api-tester/src/main/java/com/revenuecat/apitester/java/PurchasesAPI.java

```diff
@@ -160,11 +160,11 @@ static void checkLogHandler() {
         final LogHandler handler = Purchases.getLogHandler();
     }
 
-    static void check(final BillingFeature feature) {
-        switch (feature) {
-            case SUBSCRIPTIONS:
-            case SUBSCRIPTIONS_UPDATE:
-            case PRICE_CHANGE_CONFIRMATION:
-        }
-    }
+//    static void check(final BillingFeature feature) {
```

Comment:
commenting these out until i create separate api testers, otherwise CI will fail because it's not exhaustive for BC4